internal/core/src/query/visitors/ExecPlanNodeVisitor.cpp

```diff
@@ -73,6 +76,62 @@ empty_search_result(int64_t num_queries, SearchInfo& search_info) {
     return final_result;
 }
 
+void
+ExecPlanNodeVisitor::ExecuteExprNodeInternal(
+    const std::shared_ptr<milvus::plan::PlanNode>& plannode,
+    const milvus::segcore::SegmentInternalInterface* segment,
+    BitsetType& bitset_holder,
+    bool& cache_offset_getted,
+    std::vector<int64_t>& cache_offset) {
+    bitset_holder.clear();
+    LOG_SEGCORE_INFO_ << "plannode:" << plannode->ToString();
```

Comment:
I think it can using info level ,because according test, this have no heavy cost for search  and  it can help positioning problem.  this is also not frequent.